src/SoftRobots/component/constraint/model/CableModel.h

```diff
@@ -118,6 +132,18 @@ class SOFA_SOFTROBOTS_API CableModel : virtual public SoftRobotsConstraint<DataT
     Data<Real>                  d_cableInitialLength;
     Data<Real>                  d_cableLength;
 
+    Data<sofa::helper::OptionsGroup>        d_method;
```

Comment:
Could you add the description of the Data as a comment just after its declaration. See any Data declaration in SOFA